### DISCUSSION TITLE
How to initiate stress tensor for foam material?

### DISCUSSION THREAD

**Discussion by halfwitbm** [OpenRadioss/OpenRadioss#2027] [Q&A] [answered]
Hi All,

In LS-DYNA we have *MAT_LOW_DENSITY_FOAM card, *INITIAL_FOAM_REFFERENCE_GEOMETRY card, and *DEFINE_CURVE card, combine them we can initiate the initial stress tensor for foam.  Does Radioss has related cards which can do the same thing? 

Another question is, for LS-DYNNA input, it looks like OpenRadioss know cards aforementioned. OpenRadioss will read and convert LS-DYNA object into Radioss object, so read and initialize step high possible have finished in Starter. Does somebody know these processes and give me some information on which objects they convert to? especially whether the initialized stress tensor was stored in SIG array in EOS?  

Please share your knowledge, thanks.

Regards,

Bin

**Comment by PaulAltair** [comment]
I don't know about the EOS Sig array (someone else may comment!), the stress tensor is generated from ref Geom, and material entry in Radioss also.

The equivalent to Dyna initial reference geometry for foam is the Radioss /XREF card, the Dyna reference geom is mapped to that

You can check the mapping for most Dyna entries directly in the help for Commercial Radioss [Initial Ref Geom in Help](https://help.altair.com/hwsolvers/rad/topics/solvers/rad/initial_foam_reference_geometry_lsdyna_r.htm)

**Reply by PaulAltair** [reply, marked as answer]
And, similarly, for Mat low density foam: Radioss law 90

[Low density foam in help](https://help.altair.com/hwsolvers/rad/topics/solvers/rad/mat_057_low_density_foam_lsdyna_r.htm)

And Define Curve: Maps to Funct and move funct

[Define Curve in Help](https://help.altair.com/hwsolvers/rad/topics/solvers/rad/define_curve_lsdyna_r.htm)

**Reply by halfwitbm** [reply]
Hi Paul,

Thank you very much for the information. It is what I am looking for. 

Regards,

Bin

**Reply by halfwitbm** [reply]
Hi Paul,

Could you give me a small case file which use /XREF card if you have. I have searched related keywords on Goole but I didn't find such case file. Thanks.

Regards,

Bin
